infra/consumer/main.tf

```diff
@@ -0,0 +1,22 @@
+#make bucket
+resource "aws_s3_bucket" "browser" {
+  bucket_prefix = "dcp-consumer-${var.DCP_DEPLOYMENT_STAGE}"
+  acl    = "public-read"
+  force_destroy = true
+  server_side_encryption_configuration {
+    rule {
+      apply_server_side_encryption_by_default {
+        sse_algorithm     = "AES256"
+      }
+    }
+  }
+  tags = {
+    Environment = "${var.DCP_DEPLOYMENT_STAGE}"
+  }
+}
+
+#TODO make policy
```

Comment:
Just to start good practices, every time we have a TODO, could you please create a ticket in the repo that is tracked and then link the ticket in the TODO itself?

So format would look like:
`# TODO(https://github.com/chanzuckerberg/dcp-prototype/pull/9): Make policy.`